README.md

```diff
@@ -19,21 +19,26 @@ level of backwards compatibility to the official releases.
 
 **Pull requests with unofficial bug fixes and security patches from the community are encouraged and welcome!**
 
-Though Magento does not follow [Semantic Versioning](http://semver.org/) we aim to provide a workable system for
+### Versioning
+
+Though Magento does __not__ follow [Semantic Versioning](http://semver.org/) we aim to provide a workable system for
 dependency definition. Each Magento `1.<minor>.<revision>` release will get its own branch (named `1.<minor>.<revision>.x`)
 that will be independently maintained with upstream patches and community bug fixes for as long as it makes sense
-to do so (based on available resources). For example, Magento version `1.9.3.4` was merged into the `1.9.3.x` branch.
+to do so (based on available resources). For example, Magento version `1.9.4.5` was merged into the `1.9.4.x` branch.
 
 ## Requirements
 
-- PHP 7.0+ (PHP 7.3 with OpenSSL extension strongly recommended and verified compatible) (PHP 7.4 and 8.0 are supported)
-- MySQL 5.6+ (8.0+ recommended)
-- (optional) Redis 5+ (7.x recommended, latest verified compatible 7.0.4 with 20.x)
+- PHP 7.3+ (PHP 8.0 is supported)
```

Comment:
should we add also PHP 8.1?

Reply:
I have removed because i think OM is not completly tested with 8.1

E.g. https://github.com/OpenMage/magento-lts/pull/1198#issuecomment-1236416427

Maybe we should discuss it there: https://github.com/OpenMage/magento-lts/discussions/2253

Reply:
hmm ok, i said it since in composer.json we already have "php": ">=7.0 <8.2",

Reply:
hmmm, .... this should be discussed. (composer.lock is for php8.1.)

Reply:
I'm already working on a multi-part PR for PHP 8.1 compatibility (continuing from @luigifab's previous PR), part 1 should be done this week.